### DISCUSSION TITLE
When forking an upstream repository, git replace refs are not fetched

### DISCUSSION THREAD

**Discussion by nomennescio** [community/community#10404] [Pull Requests]
When forking an upstream repository, git replace refs are not fetched. Manually, you normally fetch these explicitly when cloning a repo, or you edit your local .git/config. That is not possible for a forked repo; you can only fetch new updates from the upstream, but git replace objects are ignored. Please support git replace objects.

**Comment by Mart-Bogdan** [comment]
I agree! This would be great.

Also git UI doesn't show replaced commits, but originals, sadly.
